ext/phar/tests/phpinfo_004.phpt

```diff
@@ -47,7 +47,7 @@ Phar based on pear/PHP_Archive, original concept by Davey Shafik.<br />Phar full
 <tr><td class="e">phar.require_hash</td><td class="v">Off</td><td class="v">Off</td></tr>
 </table>
 %a
-<h2><a name="module_phar">Phar</a></h2>
+<h2><a name="module_phar" name="module_phar">Phar</a></h2>
```

Comment:
I believe this change is incorrect and the test is likely skipped on GitHub actions.

Reply:
This was the case indeed, thank you for pointing this out.
I added a new test (and force-pushed) that checks with this change in focus, and made sure passes:
![image](https://user-images.githubusercontent.com/811553/178104000-307c5177-4a26-466a-8d51-0e5e8e9e00c6.png)

Reply:
> This was the case indeed, thank you for pointing this out.

The line is unmodified. Did you forget to `git add` the updated test to the commit?

Reply:
Embarrassingly, yes. Thank you for your patience with this.  Double checked and pushed _with_ those changes in this test.